index.js

```diff
@@ -0,0 +1,80 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const PORT = 3000;
+const tasksData = require("./data/tasks.json");
+const fs = require("fs");
+
+const app = express();
+
+app.use(bodyParser.json())
+
+app.listen(PORT, (error) => {
+    if (error) {
+        console.log(error)
+    }
+    else {
+        console.log(`Server running at ${PORT} successfully...`)
+    }
+})
+
+app.get("/tasks", (request, response) => {
+    return response.status(200).json({ data: tasksData });
+})
+
+app.get("/tasks/:id", (request, response) => {
+    const { id } = request.params;
+    const filteredTask = tasksData.find((task) => (parseInt(task.id) === parseInt(id)));
+    console.log(filteredTask)
+    return response.status(200).json({ data: filteredTask })
+    
+})
+
+app.post("/tasks", (request, response) => {
+    const { title, description, priority, completed } = request.body;
```

Comment:
There should be some validations, like data type checks and required parameters check